app/views/devise/sessions/new.html.erb

```diff
@@ -11,39 +11,18 @@
 </div>
  
 <div class="header_image">
-  
 </div>  
 
 <div class="log_in_box">
-  
-  <div class="box_tab">
-    LOG IN
-  </div>
-
-  <div class="form_box">
-    <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-      <div class="">
-        <%= f.email_field :email, autofocus: true, placeholder: "Username or Email" %>
-      </div>
-
-      <div >
-        <%= f.password_field :password, autocomplete: "off", placeholder: "Password"  %>
-      </div>
-
-      <div class="form_buttons">
-        <div class="remember_button">
-        <% if devise_mapping.rememberable? -%>
-            <%= f.check_box :remember_me %>
-            <%= f.label :remember_me %>
-        <% end -%>
-        </div>
-        <div class="form_buttons">
-          <%= f.submit("Log in", class: 'selectButton') %>
-        </div>
-      </div>
+  <% if devise_mapping.omniauthable? %>
+    <% resource_class.omniauth_providers.each do |provider| %>
+      <%
+        display_name = camelize_str(provider);
+        if provider == :urs
+          display_name = "Earthdata Login"
+        end
+      %>
+      <%= link_to "Login with #{display_name}", omniauth_authorize_path(resource_name, provider) %><br/>
```

Comment:
Sorry if I wasn't clear, I thought you might want to move the whole `display_name` logic (i.e. move lines 19 - 24 into the helper), and on line 25 have the `link_to "Login with #{display_name(provider)}"`...
I might also try to move the whole iterating through providers logic into the helper, but I'm not sure if that's feasible.